src/lib/dashboard-stack.ts

```diff
@@ -334,13 +336,24 @@ export class TransactionDashboardStack extends NestedStack {
       index: 'gen.py',
       runtime: Runtime.PYTHON_3_8,
       environment: {
+        INFERENCE_ARN: String(props.inferenceArn),
         QUEUE_URL: props.queue.queueUrl,
         DATASET_URL: getDatasetMapping(this).findInMap(Aws.PARTITION, IEEE),
       },
       timeout: Duration.minutes(15),
       memorySize: 3008,
     });
     props.queue.grantSendMessages(tranSimFn);
```

Comment:
remove this useless permission

Reply:
it is the FunctionName for the boto3 inside to invoke lambda?

Reply:
removed useless sqs permission